portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/Usage.path

```diff
@@ -0,0 +1,31 @@
+<html>
+<head>
+<title>Usage</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">Usage</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>PLAN_NAME</td>
+	<td>//h4[contains(.,'Plans')]/..//div[contains(@class,'plan-name') and contains(.,'${key_planName}')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>USAGE_PLAN_LIMITS_TABLE</td>
+	<td>//div[@class='limits']//ul//li[1]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>USAGE_PLAN_ADDONS_TABLE</td>
+	<td>//div[@class='addons']//ul//li[1]</td>
+	<td></td>
```

Comment:
Hi @AfonsoHSantos , I would suggest you update these paths to validate the values.